tests/tensor/test_max_argmax.py

```diff
@@ -0,0 +1,693 @@
+import builtins
```

Comment:
Is this a completely new test file? We shouldn't need a specialized file just for these Ops

Reply:
No since this is a draft PR, I included a separate test file for this function in the beginning for the purpose of ease. All the changes here will be merged in the corresponding tests in ```test_math.py``` .

Reply:
Sure!